flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
+}
```

Comment:
If the end result is x86_64 only why are there aarch64 sources around ? Like OpenJFX

Reply:
Actually, I just noticed that the OpenJFX with aarch64 is faulty and should be removed.  We are using Java 17 which is required due to some built libraries being shared between the desktop and android compiled app and Java 17 is the correct version to use with the Android Plugin for Gradle.  There are no builds of OpenJFX for Java 17 and aarch64.

Reply:
Removed incorrect aarch64 reference to just documenation.